bot/exts/recruitment/talentpool/_review.py

```diff
@@ -28,14 +26,18 @@
 
 log = get_logger(__name__)
 
-# Maximum amount of days before an automatic review is posted.
-MAX_DAYS_IN_POOL = 30
-
 # Maximum amount of characters allowed in a message
 MAX_MESSAGE_SIZE = 2000
 # Maximum amount of characters allowed in an embed
 MAX_EMBED_SIZE = 4000
 
+# Maximum number of active reviews
+MAX_ONGOING_REVIEWS = 4
```

Comment:
Are we sure we want this to be 4? 3 is the number we're currently using.

Reply:
I picked that off the top of my head thinking something like "3 for people to vote on, and 1 waiting to be actioned", and wanted to avoid slower reviews holding others up, but i'm happy to change it to 3. Can always increase depending on how it works out.

Reply:
Yeah just worried the earliest review will get buried or that the team will get saturated with reviews. Thinking about it I would maybe add a cooldown time after resolving a vote too (whether there's helpering or not), so that we don't get pinged right after resolving a vote.